Q3.txt

```diff
@@ -0,0 +1,46 @@
+Javascript:
+
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
+var x=0;
+var c=[];
+var d=[];
+var z=[];
+while (x<b.length){
+	if(b[x]%2 !==0)
+	{c[x]=b[x]*2;
+	console.log(c[x]);
+	}
+	if (b[x]%2 ===0)
```

Comment:
@aqattan  You can use an else here since an element is either odd or even, and it cannot be both. If you have two if statements then the program will ALWAYS check both statements. If you have if/else, then the program will ONLY check the second statement if it needs to. This is better performance.

Also, try to use the brackets nicely:

if (....) {
   if (...) {
   
   }
}